recipes/aiopeewee/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "aiopeewee" %}
+{% set version = "0.4.1" %}
+{% set sha256 = "03578a1ce28879f5e575c82fce8ead74a0e5ae0f7e2d2ac592f86207bf0be85e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py<36]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pytest-runner
```

Comment:
I see this is in the `setup_requires`, but it doesn't look like it's actually needed?

Reply:
The build has failed [previously](https://circleci.com/gh/kszucs/staged-recipes/38?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)

Reply:
Ah right well, if it's in `setup_requires` of course it'll try to download if not. Oops. :)